tests/test_pad_collation.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from typing import List, Tuple
+
+import numpy as np
+from parameterized import parameterized
+
+from monai.data.utils import pad_list_data_collate
+from monai.transforms import RandRotate90d, RandRotated, RandSpatialCropd, RandZoomd
+from monai.utils import set_determinism
+
+set_determinism(seed=0)
```

Comment:
is it possible to move this into `setUp`? e.g. https://github.com/Project-MONAI/MONAI/blob/db831ae096bec76371bb8c2e3db226d947a5ac06/tests/test_bounding_rect.py#L28-L32

Reply:
Done!